tests/sentry/api/endpoints/test_group_ai_summary.py

```diff
@@ -15,15 +15,18 @@ def _get_url(self, group_id: int):
         return f"/api/0/issues/{group_id}/summarize/"
 
     @patch("sentry.api.endpoints.group_ai_summary.GroupAiSummaryEndpoint._call_seer")
-    def test_ai_summary_get_endpoint_with_existing_summary(self, mock_call_seer):
+    @patch("sentry.api.endpoints.group_ai_summary.cache")
+    def test_ai_summary_get_endpoint_with_existing_summary(self, mock_cache, mock_call_seer):
```

Comment:
Personally I'd recommend not mocking the cache, but instead just store the value here using cache.set

Reply:
changed the test cases to all use the real cache